sql/moz-fx-data-shared-prod/telemetry_derived/feature_usage_v2/query.sql

```diff
@@ -477,6 +510,7 @@ activity_stream_sessions AS (
     MAX(user_prefs & 2 = 0) AS activitystream_reported_topsites_off,
     MAX(user_prefs & 4 = 0) AS activitystream_reported_pocket_off,
     MAX(user_prefs & 8 = 0) AS activitystream_reported_highlights_off,
+    MAX(user_prefs & 32 = 0) AS activitystream_reported_sponsored_topstories_off,
```

Comment:
iiuc, if topstories is turned on at during any of the sessions on the submission_date, then it would be register as true. it's not the final state of the day?

Reply:
Yeah I think that's currently the case with these prefs because of the `MAX`.. We could take the last ping value instead if you think that will be a more accurate representation. cc @irrationalagent who I think uses this dataset

Reply:
I created https://github.com/mozilla/bigquery-etl/issues/2517 to track this